qutebrowser/browser/qtnetworkdownloads.py

```diff
@@ -490,14 +490,17 @@ def _fetch_request(self, request, *, qnam=None, **kwargs):
 
     @pyqtSlot('QNetworkReply')
     def fetch(self, reply, *, target=None, auto_remove=False,
-              suggested_filename=None, prompt_download_directory=None):
+              suggested_filename=None, prompt_download_directory=None,
+              force_overwrite=False):
         """Download a QNetworkReply to disk.
 
         Args:
             reply: The QNetworkReply to download.
             target: Where to save the download as downloads.DownloadTarget.
             auto_remove: Whether to remove the download even if
                          downloads.remove_finished is set to -1.
+            force_overwrite: Whether to overwrite the target without
+                             prompting, for supported targets.
```

Comment:
Hmm, makes me wonder whether it'd make more sense to attach this to the `FileNameTarget` class?